spec/posthog/feature_flag_spec.rb

```diff
@@ -978,6 +978,45 @@ class PostHog
       expect(FeatureFlagsPoller.match_property(property_d, { 'key' => 44 })).to be false
 
     end
+
+    it 'with date operators' do
+      # is date before
+      property_a = { 'key' => 'key', 'value' => '2022-05-01', 'operator' => 'is_date_before'}
+      expect(FeatureFlagsPoller.match_property(property_a, { 'key' => '2022-03-01' })).to be true
+      expect(FeatureFlagsPoller.match_property(property_a, { 'key' => '2022-04-30' })).to be true
+      expect(FeatureFlagsPoller.match_property(property_a, { 'key' => DateTime.new(2022, 4, 30)})).to be true
+      expect(FeatureFlagsPoller.match_property(property_a, { 'key' => DateTime.new(2022, 4, 30, 1, 2, 3)})).to be true
+      expect(FeatureFlagsPoller.match_property(property_a, { 'key' => DateTime.new(2022, 4, 30, 1, 2, 3, '+2')})).to be true
+      expect(FeatureFlagsPoller.match_property(property_a, { 'key' => DateTime.parse('2022-04-30')})).to be true
+      expect(FeatureFlagsPoller.match_property(property_a, { 'key' => '2022-05-30'})).to be false
+
+      # error handling
+      expect(FeatureFlagsPoller.match_property(property_a, { 'key' => 1})).to raise_error(InconclusiveMatchError)
```

Comment:
error handling works but the tests keep failing for it... 😕

Reply:
You need curly braces here. Subtle but important difference ->

https://github.com/PostHog/posthog-ruby/blob/c5055220ad81055568d86dbf4bd8cc110b453655/spec/posthog/feature_flag_spec.rb#L815

This executes is as a block, which allows wrapping & collecting any error that raises, while your version is immediately executed, thus fails.

Reply:
also FYI eric asked me to help out on updating the ruby/node libs with the date operators, hope that's alright with you!

Reply:
For sure! *logs off for real this time*